frontend/lib/resolution/call-init-deinit.cpp

```diff
@@ -788,8 +788,8 @@ void CallInitDeinit::handleDeclaration(const VarLikeDecl* ast, RV& rv) {
   VarFrame* frame = currentFrame();
 
   // check for use of deinited variables in type or init exprs
-  if (auto init = ast->initExpression()) {
-    processMentions(init, rv);
+  if (auto type = ast->typeExpression()) {
+    processMentions(type, rv);
   }
```

Comment:
wow, that was unexpected (I see the copy of `init = ` code below.

Reply:
which code is copied?